### ISSUE TITLE
Classifier service parent-filter

### ISSUE THREAD

**Issue by petermezei** [slamby/slamby-insight#24]
Built-in classifier service recommend filter option. Filter by tags during the recommendation

**Comment by attilalaszlo** [comment]
ready: #35

**Comment by petermezei** [comment]
I tested it with the agri environment. I used the classifier service with alias classifier. I tried the service with "MTZ 82 traktor" without any parent tag filter. The first result category id is: 6761.

Now I did it again, but with Parent tag filter. I filtered the tags to 6761 and was looking for the parent tag. It was 3118. Now I was looking for the 3118 as a parent category and selected it.

Now click the Ok, but error message: `There is at least one parent tag in the ParentTagIdList which is not a parnet of any activated tags!`

@attilalaszlo  could you check it? It seems to be a bug.

**Comment by attilalaszlo** [comment]
1. Did you copy the error message or type it directly? Because there is a typo and I want to fix it if it's in the API

2. Can you please try out with a new classifier service? I'm afraid that service doesn't contain the parent tag lists metadata next to the dictionaries

**Comment by petermezei** [comment]
1. Checked it and I typed it wrongly.
2. I tried it out, but the same error. On demo server I created a parent-tag tag in the e-commerce dataset, and added Home category as a child. Then I created a new classifier service, activated it, then I tried the same. The test text: `home kitchen design lamp`

Error print screen:
![image](https://cloud.githubusercontent.com/assets/18461114/23163991/58e6ebdc-f835-11e6-9e40-585f5f81f5b7.png)
